app/build.gradle

```diff
@@ -447,6 +451,9 @@ if (project.hasProperty("coverage")) {
     }
 }
 
+
+apply from: 'https://github.com/mozilla-mobile/android-components/raw/master/components/service/glean/scripts/sdk_generator.gradle'
```

Comment:
I'm not sure what this does but I wonder if this is necessary: I'd think the glean team would come up with a better way to apply this.

That being said, if this is the recommended way, we should consider an alternative like downloading a copy of this and putting it in the repository. Applying scripts from the internet creates issues: it prevents reproducible builds and leaves developers/the app more open to vulnerabilities if this script changes and we're unaware of it.

Reply:
The optimal way according to Travis is like Fenix, see here: https://github.com/mozilla-mobile/fenix/blob/3253f01667b2b0757614880138c85fd94b8c357d/app/build.gradle#L464 I am currently using the same one as Reference Browser https://github.com/mozilla-mobile/reference-browser/blob/0a9d541b8f343914c3e10f13cc6b13369050dbc1/app/build.gradle#L297 also shown to me by Travis but he recommends changing it so it is using the version tag of AC that FFTV is using.